src/nmark/def.nim

```diff
@@ -177,21 +177,19 @@ const
   reLinkRef = re" {0,3}\[\s*.*\s*]:(\s*\n?\s*)"
   reEntity* = re"&[a-zA-Z0-9#]+;"
 
-const olNum* = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
-const puncChar* = ['!', '"', '#', '$', '%', '&', '\'', '(', ')', '+', ',', '-', '.', '/', ':', ';', '<', '=', '>', '?', '@', '[', '\\', ']', '^', '`', '{', '|', '}', '~']
-const unchangedChar* = ['!', '#', '$', '%', '\'', '(', ')', '+', ',', '-', '.', '/', ':', ';', '=', '?', '@', '[', '\\', ']', '^', '`', '{', '|', '}', '~', '_', '*']
+const olNum* = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9'}
```

Comment:
These 3 are kind of interesting - so, in Nim default builtin sets are implemented as bit vectors, so checking if a char is in a set is faster than checking if a char is in an array.